tests/js/spec/components/dropdownLink.spec.jsx

```diff
@@ -131,7 +131,10 @@ describe('DropdownLink', function () {
       });
 
       it('does not close when document is clicked', function () {
-        $(document).click();
+        const evt = document.createEvent('HTMLEvents');
```

Comment:
should we name this const event too? just to be consistent :D

Reply:
nah...I just noticed that in this test everything is evt

Reply:
yeah 🤷